service/service.py

```diff
@@ -108,6 +106,41 @@ def internal_server_error(error):
                    error='Internal Server Error',
                    message=message), status.HTTP_500_INTERNAL_SERVER_ERROR
 
+######################################################################
+# Configure Swagger
+######################################################################
+api = Api(app,
+          version='1.0.0',
+          title='Wishlists REST API Service',
+          description='This is the Wishlists service.',
+          default='wihslists',
+          default_label='Wishlist shop operations',
+          doc='/apidocs', 
+          authorizations=authorizations
```

Comment:
we don't need authorizations for the application

Reply:
Ok, removed.

Reply:
thanks